peps/api/index.rst

```diff
@@ -0,0 +1,64 @@
+PEPs API
+========
+
+There is a read-only API of published PEPs available at:
+
+* https://peps.python.org/api/peps.json
+
+The structure is like:
+
+.. code-block:: json
+
+   {
+     "0": {
```

Comment:
These aren't great examples because they're all very old and special (e.g. there's a number of fields that are `null` on all of them). Might be better to give examples that demonstrate more of the fields. (I think it's OK if we don't cover all of `requires`, `replaces`, and `superseded_by`, since those are, but we should get the other ones.)

Suggestion: use 8, 484 (two of the most well-known PEP numbers), and 622 (demonstrates superseded_by).